Classes/Service/OpenIdConnectService.php

```diff
@@ -32,7 +34,33 @@ public function __construct(OAuthService $OAuthService, array $config = [])
         $this->config = $config ?: GeneralUtility::makeInstance(ExtensionConfiguration::class)->get('oidc') ?? [];
     }
 
-    public function generateAuthenticationContext(ServerRequest $request, array $authorizationUrlOptions = []): AuthenticationContext
+    public function isAuthenticationRequest(ServerRequestInterface $request): bool
+    {
+        $language = $request->getAttribute('language');
+        return $request->getUri()->getPath() === $language->getBase()->getPath() . $this->config['authenticationUrlRoute'];
```

Comment:
There's no fallback if this config is not set (update without going back to extension setup) but that's not a big deal, it will be a major version anyway... :)

Reply:
No need to make this a major version. ;-)
Actually on ext update, this should be updated by core. Other than that I'm fine adding a fallback here, no problem.